sanity-io/structured-content-2022
# Update conference logo

# Update conference logo

## Intent

Update the conference logo, as per Sindre's comment in 
https://app.shortcut.com/sanity-io/story/15376/updated-design-elements-consider-if-and-how-they-effect-the-brand#activity-15925

I filed a separate story to track it: https://app.shortcut.com/sanity-io/story/15936/update-conference-logo

## Description

Just a copy/paste replace from Slack.

## Testing this PR
1. Open the [front page](https://structured-content-2022-web-git-update-conference-logo.sanity.build/)
2. Verify that the big logo at the top has less space between the lines than it does on staging
3. Scroll down until the header bar appears and verify that the logo is similarly changed there
